frontend/src/components/GameList.vue

```diff
@@ -1,6 +1,11 @@
 <template>
-<div v-for = "item in [0,1,2,3,4,5,6,7]">
-  <GameCard :title="'Game Title'" :author="'Game Author'" :description="'Game Description'" image="https://cdn.cloudflare.steamstatic.com/steam/apps/489830/capsule_616x353.jpg?t=1650909796" />
+<div v-for = "item in gameInfo">
+  <GameCard 
+    :title="'title'" 
```

Comment:
So this needs to be `:title="item.title"` and you need to do a similar thing for the ones below.